docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -903,6 +905,15 @@ sources described above to create conditional samplers for first name
 and last name based on soundex.  Perhaps measure of success is to look
 at entropy of character n-gram distribution.
 
+To simulate naming after a parent or family member, have 5% of children 
+born in the simulation be named after a parent that matches their 
+gender. If there is not a parent that matches their gender, or that 
+parent is not tracked, they will have the name from another relative 
```

Comment:
We may not know who in the household is a relative of this simulant, if the person who gives birth to them is not a relative of the reference person. I don't think what you've written here is incorrect, but you might want to spell this out more explicitly.

My understanding: 5% chance -> if tracked parent of same sex, assign that first name -> if relative of ref person and there is another relative of ref person of the same sex, assign that first name -> assign a random first name.

Reply:
Yeah, this looks cleaner. I made the text follow this flow

Reply:
I agree with Zeb

At the moment, the only defined "parents" that exist in the sim are the mothers of childs born in the simulation.

If we want 5% of children to take on a parent's name, and you guys are also planning on coming up with a way for us to assign parents, we should hold off on assigning 5% of children their parents names until you've come up with a method for assigning parents to children.

however, if you guys end up deciding you don't want to build in structure that links parents and children, we'll want a more specific set of rules for this. e.g.:
- for anyone in a household with the relationship to head of household "biological child" or "adopted child", if they share the sex of the "Reference person", then have a 5% chance of getting the same first name as the reference person
^ though this will miss a lot of the parent-child dyads you were hoping to catch